torch/testing/_internal/common_methods_invocations.py

```diff
@@ -14546,9 +14546,11 @@ def ref_pairwise_distance(input1, input2):
             # On CUDA, the op is dispatched (and a few more conditions) to
             # _fused_dropout, which doesn't support forward AD
             DecorateInfo(unittest.skip("Skipped!"), 'TestGradients', 'test_forward_mode_AD', device_type='cuda'),
-            # NotImplementedError: Trying to use forward AD with native_dropout that does not support it
-            DecorateInfo(unittest.expectedFailure, 'TestGradients', 'test_fn_fwgrad_bwgrad',
-                         device_type='cuda', dtypes=[torch.float64]),),
+            # (ROCm) NotImplementedError: Trying to use forward AD with native_dropout that does not support it
```

Comment:
Is `(ROCm)` implies that this test should be skipped only for ROCm and it works on CUDA? 
I don't think we should change `unittest.expectedFailure` -> `unittest.skip("Skipped!")`.

Reply:
Thanks for pointing this out. It was a skip before from [this](https://github.com/pytorch/pytorch/pull/70061/files#diff-18a0c3e7cd07089b34760028c387d39de1aa393172f1d5f123692106b3a3d144R14561-R14563) PR and was changed to `expectedFailure` [here](https://github.com/pytorch/pytorch/pull/71944). I think I missed something during the rebase. I'll fix this now.